packages/rest-client-jsonapi/src/services/jsonapi/jsonapi.ts

```diff
@@ -35,6 +35,17 @@ export function jsonapi<
   // Default `type` to `schemaMap` key if not set in `schemaMap`
   const completeSchemaMap = Object.entries(schemaMap).reduce(
     (acc, [key, value]) => {
+      if (value.namespace) {
+        if (key !== `${value.namespace}_${value.name}`) {
+          console.error(
+            "The schema name should be in the form of the namespace_name properties",
```

Comment:
lets make this error a bit more narrow by putting something like `the key ${key} .... ${namespace}_${name}` so they know which schema is the problem one